README.md

```diff
@@ -619,6 +627,11 @@ Power available from time-varying generator (hydro, wind, solar) ![i](https://re
 at time ![t](https://render.githubusercontent.com/render/math?math=t).
 - ![x_{l}](https://render.githubusercontent.com/render/math?math=x_%7Bl%7D):
 Impedance of branch ![l](https://render.githubusercontent.com/render/math?math=l).
+- ![\overline{\delta}_{b, t}](https://render.githubusercontent.com/render/math?math=%5Coverline%7B%5Cdelta%7D_%7Bb%2Ct%7D):
+Demand flexibility available at bus ![b](https://render.githubusercontent.com/render/math?math=b)
+at time ![t](https://render.githubusercontent.com/render/math?math=t).
```

Comment:
Are there different limits for each direction?

Reply:
Not right now. I think they should probably be different though. If this were a single consumer model, I would probably set the upper bound on `load_shift_up` to be something like the consumer's maximum demand. However, since we are dealing with demand from different sectors, where demand can vary a lot (e.g., based on weather, based on EV travel), I don't feel like the maximum nodal demand is a good metric. My feeling is that the upper bound on `load_shift_up` should be a value in between the current upper bound (i.e., the value obtained from the demand flexibility profile) and the maximum nodal demand (which I know is quite a large spread). This is something that warrants a little further exploration.

Reply:
Then should we use two different parameters to maximize the flexibility for the sake of future usage and define them to be equal at this point. It could be simply noted down here in the readme without modifying the code for now to avoid complaints of redundant parameters (?).

Reply:
Yeah, good idea. I will update the documentation

Reply:
Done.